workflow/type_test.go

```diff
@@ -0,0 +1,81 @@
+package workflow
+
+import (
+	"testing"
+
+	"github.com/mesg-foundation/engine/event"
+	"github.com/mesg-foundation/engine/hash"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMatch(t *testing.T) {
+	var tests = []struct {
+		trigger *trigger
+		event   *event.Event
+		match   bool
+	}{
+		{ // matching event
+			&trigger{InstanceHash: hash.Int(1), EventKey: "xx"},
```

Comment:
Just a small fix, but do not use magic names/values etc in test (like xx :)), rather put `-` or `(none)`  for other developers

Reply:
in fact, just "-" or "(none)" doesn't cover all the test cases and it is even harder to read for the complex tests like the ones with the filters.
If it's not blocking I would keep it like that or otherwise, we need at least 4 or 6 magic but conventional values...